Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -222,6 +222,25 @@ static int SendFrameAndReceivePacket(AVCodecContext* avctx, AVPacket* pkt, AVFra
 #endif
 }
 
+static void WritePacket(AVPacket& pkt)
```

Comment:
This function can take `codec_context`, `s_stream`, and `s_format_context` as arguments, similarly to `RecievePacket`